apps/auth/auth.py

```diff
@@ -108,6 +108,9 @@ def check_permissions(self, resource, method, user):
         if not user:
             return True
 
+        if resource == '_blueprint':
```

Comment:
Can you add some comments into `Step 1` in this function's docstring explaining this auth condition

Reply:
Added